wger/nutrition/tests/test_sync.py

```diff
@@ -101,9 +101,9 @@ def test_ingredient_sync(self, mock_request):
         self.assertEqual(ingredient.code, '1234567890987654321')
 
         # Act
-        sync_ingredients(lambda x: x)
+        sync_ingredients(lambda x: x, language_codes='en')
         mock_request.assert_called_with(
-            'https://wger.de/api/v2/ingredient/?limit=999',
+            'https://wger.de/api/v2/ingredient/?limit=999&language=2',
```

Comment:
I'd add a new simple test here, something like `def test_ingredient_sync_languages` that only checks that the correct URL is called